public_html/wp-content/plugins/wc-post-types/inc/deprecated.php

```diff
@@ -0,0 +1,616 @@
+<?php
+/**
+ * Deprecated functionality.
+ *
+ * This is still loaded on all WordCamp sites, but is considered unsupported. Instead of using shortcodes to
+ * display content, the corresponding blocks should be used.
+ */
+
+class WordCamp_Post_Types_Plugin_Deprecated {
```

Comment:
This may not currently be anywhere in our dev docs, but we've been trying to move away from old-school namespacing via singleton classes. The [privacy.php](https://github.com/WordPress/wordcamp.org/blob/production/public_html/wp-content/plugins/wc-post-types/inc/privacy.php) and [rest-api.php](https://github.com/WordPress/wordcamp.org/blob/production/public_html/wp-content/plugins/wc-post-types/inc/rest-api.php) files are examples of starting to do that in this plugin.

- [ ] `[SHOULD]` Use something like `WordCamp\Post_Types\Deprecated` as a namespace instead of having everything as methods of a class.

Reply:
Good to know– I saw the pattern but didn't make that connection :) I've updated the PR.